static/app/views/issueList/overviewFc.tsx

```diff
@@ -0,0 +1,1237 @@
+    }
+  }, [realtimeActive, resumePolling]);
+
+  // Fetch data on mount if necessary
+  useEffect(() => {
+    const loadedFromCache = loadFromCache();
+    if (!loadedFromCache) {
+      // It's possible the projects query parameter is not yet ready and this
+      // request will be repeated in componentDidUpdate
+      fetchData();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const previousSelection = usePrevious(selection);
+  const previousSavedSearchLoading = usePrevious(savedSearchLoading);
+  const previousIssuesLoading = usePrevious(issuesLoading);
+
+  const previousRequestParams = usePrevious(requestParams);
+
+  // Keep data up to date
+  useEffect(() => {
+    const selectionChanged = !isEqual(previousSelection, selection);
+
+    // Wait for saved searches to load before we attempt to fetch stream data
+    // Selection changing could indicate that the projects query parameter has populated
+    // and we should refetch data.
+    if (savedSearchLoading && !selectionChanged) {
+      return;
+    }
+
+    if (previousSavedSearchLoading && !savedSearchLoading) {
+      return;
+    }
+
+    // If any important url parameter changed or saved search changed
+    // reload data.
+    if (!isEqual(previousRequestParams, requestParams)) {
+      fetchData(selectionChanged);
+    } else if (
+      !lastRequestRef.current &&
+      previousIssuesLoading === false &&
+      issuesLoading
+    ) {
+      // Reload if we issues are loading or their loading state changed.
+      // This can happen when transitionTo is called
+      fetchData();
+    }
+  }, [
+    fetchData,
+    savedSearchLoading,
+    selection,
+    previousSelection,
+    organization.features,
+    issuesLoading,
+    loadFromCache,
+    previousSavedSearchLoading,
+    previousIssuesLoading,
+    previousRequestParams,
+    requestParams,
+  ]);
+
+  // Fetch members on mount
+  useEffect(() => {
+    const projectIds = selection.projects.map(projectId => String(projectId));
+
+    fetchOrgMembers(api, organization.slug, projectIds).then(members => {
+      setMemberList(indexMembersByProject(members));
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  // If the project selection has changed reload the member list and tag keys
+  // allowing autocomplete and tag sidebar to be more accurate.
+  useEffect(() => {
+    if (isEqual(previousSelection?.projects, selection.projects)) {
+      return;
+    }
+
+    const projectIds = selection.projects.map(projectId => String(projectId));
+
+    fetchOrgMembers(api, organization.slug, projectIds).then(members => {
+      setMemberList(indexMembersByProject(members));
+    });
+  }, [api, organization.slug, selection.projects, previousSelection?.projects]);
+
+  // Cleanup
+  useEffect(() => {
+    return () => {
+      pollerRef.current?.disable();
+      SelectedGroupStore.reset();
+      GroupStore.reset();
+      api.clear();
```

Comment:
i think api.clear is called for you if not using persist in flight

Reply:
Yeah, removed